src/refactor/inputHandler.py

```diff
@@ -0,0 +1,11 @@
+
+
+
+
+class inputHandler():
```

Comment:
Is the purpose of this class just for logging? If so, we can probably rename it to be more clear as to what it does

Reply:
It's to log inputs typed on the terminal, and hopefully in the future allow using up arrow to scroll through previous commands. It would give the groundStation class too much responsibility to add all that functionality there, so another, tiny class is used

Reply:
Sounds good! I can also extend this class to log inputs to the website's database (optionally when a database url is provided as an env variable) so everything is logged even if an operator uses this cli.

You may also want gitignore command_history.txt so it doesn't get accidentally committed.

Reply:
I'll actually just change it to .log, which is already gitignored